fury/io.py

```diff
@@ -11,10 +11,48 @@
                       XMLPolyDataReader, PLYReader, STLReader,
                       OBJReader, MNIObjectReader, PolyDataWriter,
                       XMLPolyDataWriter, PLYWriter, STLWriter,
-                      MNIObjectWriter)
+                      MNIObjectWriter, ImageFlip, Texture)
 from fury.utils import set_input
 
 
+def load_cubemap_texture(fnames, interpolate_on=True, mipmap_on=True):
+    """Load a cube map texture from a list of 6 images.
```

Comment:
What happens if there is more or less? maybe you should check it

Reply:
If that happens VTK throws an error. I should assert that in the respective test.

Reply:
Captured in `load_cubemap_texture` and tested in `test_load_cubemap_texture` in `fury/tests/test_io.py`.